sources/packages/api/src-sql/Reports/Drop-disbursement-report.sql

```diff
@@ -0,0 +1,2 @@
+--Insert config for disbursement report.
```

Comment:
The comment seems wrong. I personally will not request a comment to be added to a so self-explanatory line 😉 
IMO it can be removed.